source/elements/oneMKL/source/domains/spblas/trmvoptimize.rst

```diff
@@ -101,7 +28,7 @@ optimize_trmv (USM version)
                                  oneapi::mkl::transpose               transpose_val,
                                  oneapi::mkl::diag                    diag_val,
                                  oneapi::mkl::sparse::matrix_handle_t handle,
-                                 std::vector<sycl::event>             &dependencies);
+                                 std::vector<sycl::event>             &dependencies = {});
```

Comment:
I think this might need a const out front as well.  right ?

Reply:
so it is a const reference

Reply:
Oh, right. I just made it with default value. I'll check and update as necessary.

Reply:
added `const` keyword for the dependencies list in those `optimize_xyz` routines.